WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs

```diff
@@ -539,4 +545,91 @@ public async Task JadeKataAsync()
 		var checkResult = signedTx.Check();
 		Assert.Equal(TransactionCheckResult.Success, checkResult);
 	}
+
+
+	[Fact]
+	public async Task BitBox02BtcOnlyKataAsync()
```

Comment:
you call it _BitBox02**BtcOnly**_
Does that mean that the Bitbox with the shitcoins firmware doesn't work/ is not supported?

otherwise the _BtcOnly_ could be removed I guess

(then it works the same as Trezor, they also have BtcOnly firmware and one with shitcoins)

Reply:
Right now, we have only the BitBox02 BtcOnly version, so I think we only can support a device that we have.

Reply:
https://bitbox.shop/en/?u=5522422611155248203215026#products

Reply:
resolved?

Reply:
Yes, I think so